docusaurus.config.js

```diff
@@ -57,7 +82,7 @@ module.exports = {
       isCloseable: true,
     },
     prism: {
-      additionalLanguages: ["java", "protobuf"],
+      additionalLanguages: ["java", "protobuf", "scala"],
```

Comment:
I don't know _why_ I had to add this to prevent the plugin from crashing. [It was an issue in previous versions](https://github.com/rohit-gohri/redocusaurus/issues/183), but it appears that the issue was resolved. 🤷

Reply:
😵 So weird but 🤷‍♀️